pandas/tests/plotting/test_frame.py

```diff
@@ -144,6 +144,12 @@ def test_plot(self):
         result = ax.axes
         assert result is axes[0]
 
+    # GH 25587
+    def test_integer_array_plot(self):
+        s = Series([4, 5, 3, 2], dtype="UInt32")
+        _check_plot_works(s.plot, yticks=[1, 2, 3, 4])
+        _check_plot_works(s.plot, xticks=[4, 5, 3, 2])
```

Comment:
Parameterize this as well e.g.

~~~python
@pytest.mark.parametrize("kwargs", [dict(yticks=[1,2,3,4]), dict(xticks=[4,5,3,2])])
def test_integer_array_plot(self, kwargs):
     s = Series(...)
     _check_plot_works(s.plot, **kwargs)
~~~

This is just to allow for slightly more modular testing.

Reply:
What's the reason for passing xticks and yticks in the first place? I don't see anything in the original issue referencing them.

Reply:
Fixed.

Checking the value of `yticks` and `yticks` here is to ensure the conversion works correctly.